PyInstaller/depend/dylib.py

```diff
@@ -92,6 +92,7 @@
     r'api-ms-win-core.*',
     r'api-ms-win-crt.*',
     r'ucrtbase\.dll',
+    r'vcruntime140\.dll',
```

Comment:
I'm not an expert on Microsoft's C Runtime, however it appears that Microsoft considers vcruntime140.dll to be part of the new Universal CRT. The following article states that the CRT is divided into two logical parts: 1) the VCRuntime (compiler support), and 2) pure libraries (which I'm guessing are the api-ms-* and ucrtbase.dll files). https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/